src/Host.cpp

```diff
@@ -369,6 +369,28 @@ void Host::resetProfile()
     qDebug()<<"resetProfile() DONE";
 }
 
+// Saves profile to disk - does not save items dirty in the editor, however.
+// returns true if successful or false+error message if not
+std::tuple<bool, QString, QString, QString, QString> Host::saveProfile()
```

Comment:
I'll add the ability to specify a custom save location. I don't like the idea of passing in an an error message argument that'll get filled in - it won't solve my problem of letting the caller decide whenever to use the translated or non-translation version of the string.